src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int firstLag;
```

Comment:
The variable name `firstLag` might be confusing. Consider renaming it to `side` for better clarity and understanding, as it represents the side length of the square.